assets/js/lib/analytics/index.js

```diff
@@ -1,22 +1,53 @@
 import posthog from 'posthog-js';
+import { v5 as uuidv5 } from 'uuid';
 
 import { getFromConfig } from '@lib/config/config';
 
-const analyticsEnabled = getFromConfig('analyticsEnabled');
+const analyticsEnabledConfig = getFromConfig('analyticsEnabled');
 const analyticsKey = getFromConfig('analyticsKey');
 const analyticsUrl = getFromConfig('analyticsUrl');
 const installationID = getFromConfig('installationID');
 
-if (analyticsEnabled) {
+if (analyticsEnabledConfig) {
   posthog.init(analyticsKey, {
     api_host: analyticsUrl,
+    opt_out_capturing_by_default: true,
     capture_pageview: false,
+    disable_persistence: true,
   });
 }
 
-export const capture = (event, payload) => {
+export const getAnalyticsEnabledConfig = () => analyticsEnabledConfig;
```

Comment:
Let's maybe put a reminder comment saying to remove this function. It only is needed because the feature is not enabled yet. Or it is fine if you remember hehe